resultstore-search-api/mock/dummy_file.html

```diff
@@ -0,0 +1,960 @@
+<!DOCTYPE html><html><head><meta charset="utf-8"><title>Multi Dimensional Table (Plx custom chart chart)</title>
```

Comment:
I don't believe you need the entire HTML. Probably cut it down to the point where your test passes.